--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,42 +46,6 @@ add_subdirectory(examples)
 # Packaging
 ##############################################################################
 
-# parse /etc/os-release
-file(READ "/etc/os-release" os_version)
-string(REPLACE "\n" ";" os_version ${os_version})
-foreach(_ver ${os_version})
-  string(REPLACE "=" ";" _ver ${_ver})
-  list(GET _ver 0 _name)
-  list(GET _ver 1 _value)
-  set(OS_${_name} ${_value})
-endforeach()
-
-  # extract version from git
-execute_process(
-  COMMAND git describe --long --match v*
-  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
-  OUTPUT_VARIABLE VER
-  OUTPUT_STRIP_TRAILING_WHITESPACE
-)
-string(REGEX REPLACE "v(.*)-([0-9]+)-(g[0-9a-f]+)" "\\1;\\2;\\3" VER ${VER})
-list(GET VER 0 tag)
-list(GET VER 1 commit_num)
-list(GET VER 2 commit_name)
-
-#define DEB and RPM version numbers
-if(${commit_num} EQUAL 0)
-  set(deb_ver "${tag}")
-  set(rpm_ver "${tag}")
-else()
-  if (DEFINED ENV{BUILD_NUMBER})
-    set(deb_ver "${tag}~${commit_num}-${commit_name}~b$ENV{BUILD_NUMBER}")
-    set(rpm_ver "${tag}~${commit_num}_${commit_name}~b$ENV{BUILD_NUMBER}")
-  else()
-    set(deb_ver "${tag}~${commit_num}-${commit_name}")
-    set(rpm_ver "${tag}~${commit_num}_${commit_name}")
-  endif()
-endif()
-
 set(CPACK_PACKAGE_NAME "memif")
 set(CPACK_STRIP_FILES OFF)
 set(CPACK_PACKAGE_VENDOR "fd.io")
-- 
2.33.1

